src/apps/loan-list/modal/renew-button.tsx

```diff
@@ -46,9 +52,7 @@ const RenewButton: FC<RenewButtonProps> = ({ loanId, faust, renewable }) => {
         type="button"
         disabled={!renewable}
         onClick={() => renew(loanId)}
-        className={`btn-primary btn-filled btn-small arrow__hover--right-small ${
-          !renewable ? "btn-outline" : ""
-        }`}
+        className={`btn-primary btn-filled btn-small arrow__hover--right-small ${classNames}`}
```

Comment:
I think you should use the `Button` component here. You wouldn't have to use all the classes then.